mne/simulation/raw.py

```diff
@@ -149,7 +153,8 @@ def simulate_raw(info, stc=None, trans=None, src=None, bem=None, head_pos=None,
         See Notes for details.
 
         .. versionchanged:: 0.18
-           Support for tuple, and iterable of tuple or SourceEstimate.
+           Support for tuple, iterable of tuple or SourceEstimate,
+           or SourceSimulator.
```

Comment:
the stc param type description should be changed accordingly. So far it just says:

`stc : iterable | SourceEstimate`